web/modules/custom/helloworld/helloworld.module

```diff
@@ -0,0 +1,29 @@
+<?php 
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * @file
+ * Implements hooks.
+ */
+
+/**
+ * Implements hook form alter.
+ *
+ * @param array $form
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The FormStateInterface object.
+ * @param string $form_id
+ * @return void
```

Comment:
This is not required when implementing hooks.
Please check some examples from core.